sky/packages/sky/lib/gestures/show_press.dart

```diff
@@ -0,0 +1,31 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:sky' as sky;
+
+import 'package:sky/base/pointer_router.dart';
+import 'package:sky/gestures/arena.dart';
+import 'package:sky/gestures/constants.dart';
+import 'package:sky/gestures/recognizer.dart';
+
+typedef void GestureShowPressListener();
+
+class ShowPressGestureRecognizer extends PrimaryPointerGestureRecognizer {
+  ShowPressGestureRecognizer({ PointerRouter router, this.onShowPress })
+    : super(router: router, deadline: kTapTimeout);
+
+  GestureShowPressListener onShowPress;
```

Comment:
We're going to want a way to cancel this. For example, suppose you have a scroll, a tap, and a showpress all in the arena together. You want to show the press from the kTapTimeout until you decide to start scrolling, at which point (when the tap is rejected) you want to stop showing the press.